server/config.ts

```diff
@@ -36,6 +36,10 @@ export interface ApiConfig {
 export default {
   https: production,
   staticResourceCacheDuration: 20,
+  environment: process.env.ENVIRONMENT || 'local',
```

Comment:
I'm guessing this is a HMPPS/MoJ standard? It seems strange to use `ENVIRONMENT` when we use `NODE_ENV` elsewhere but happy to bow to superior knowledge

Reply:
Yeah, `NODE_ENV` will usually be `production` for production like envs, and `development` otherwise. This just allows us to differentiate between `dev`, `prod` and `pre-prod`